api/swagger.yml

```diff
@@ -176,6 +176,19 @@ components:
           items:
             $ref: "#/components/schemas/Repository"
 
+    FindMergeBaseResult:
+      type: object
+      properties:
+        SourceCommitID:
+          type: string
+          description: "The commit ID of the merge source"
+        DestinationCommitID:
+          type: string
+          description: "The commit ID of the merge destination"
+        BaseCommitID:
+          type: string
+          description: "The commit ID of the merge base"
```

Comment:
- We usually specify the properties in snake case. the code generation will make them relevant to the target language as needed.
- Think the response should make all 3 as required - unless there is a valid response without one of them.

Reply:
It will be very strange that when I write a client code and get a poiner to a base commit and not the base commit for lookup for base commit.